dli_app/mod_reports/models.py

```diff
@@ -28,18 +29,16 @@ class Report(db.Model):
     )
 
     def __repr__(self):
-        # TODO: Find a way of describing this report
-        return '<Report>'
+        return '<Report %r>' % self.rep_name
 
 class Field(db.Model):
     __tablename__ = "field"
     id = db.Column(db.Integer, primary_key=True)
     field_type = db.relationship('FieldType')
-    # TODO: Fill out remaining fields
-
+    department_id = db.Column(db.Integer, db.ForeignKey("department.id"))
+    name = db.Column(db.String(32))
```

Comment:
One more thing to add to this (for easier retrieval within code):
data_points = db.relationship(
'FieldData',
backref='field',
)